libdevice/fallback-cmath.cpp

```diff
@@ -168,8 +168,13 @@ float __devicelib_fmaf(float x, float y, float z) {
   return __spirv_ocl_fma(x, y, z);
 }
 
+#if !defined(__NVPTX__) && !defined(__AMDGCN__)
```

Comment:
The condition shouldn't be inverted. For future extension any potential new target should not be assumed to be buggy.

```suggestion
#if defined(__SPIR__) || defined(__SPIRV__)
```

Reply:
Hi, @Maetveis 
Fixed.
Thanks very much.